test/e2e/run-all.sh

```diff
@@ -6,8 +6,9 @@ set -u
 set -o pipefail
 
 export PATH="$PATH:./node_modules/.bin"
-export GANACHE_ARGS='--quiet --blockTime 2'
+export GANACHE_ARGS="${GANACHE_ARGS:---quiet}"
 
+export GANACHE_ARGS="--blockTime 2 $GANACHE_ARGS"
```

Comment:
Can this line be combined with line 9? e.g.
```suggestion
export GANACHE_ARGS="--blockTime 2 ${GANACHE_ARGS:---quiet}"
```

It just looks a bit odd to see the same environment variable overriding itself immediately after being set.

Reply:
I separated them explicitly because the `--blockTime 2` is required for tests to function and I don't think that should necessarily leak out. For example, setting `GANACHE_ARGS=''` should still cause the tests to pass, I'd think.

Reply:
And we'd need to inline this 3x to remove line 3

Reply:
That would still be the case with the change I suggested though. I'm not sure functionally what the difference would be, or what you mean when you say "leak out".

Reply:
> That would still be the case with the change I suggested though

Yup, I added https://github.com/MetaMask/metamask-extension/pull/6870#discussion_r304601086 after as well, we'd need to inline it a few times to be able to remove line 9. Should we inline it everywhere instead?

Reply:
I don't follow why you think it would need to be inlined.

In fact, as-is the `blockTime 2` flag is getting repeated. It's there twice for the 2nd call to concurrently, and 3 times for the last one.

Reply:
Oh shoot, yes. I'm misunderstanding my own PR. Hmm, this isn't quite what I meant to achieve.